src/main/java/org/terasology/climateConditions/BodyTemperatureComponent.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.climateConditions;
+
+import org.terasology.entitySystem.Component;
+import org.terasology.network.Replicate;
+
+public class BodyTemperatureComponent implements Component {
+
+    @Replicate
+    public float bodyTemperature = 0.4f;
```

Comment:
Nitpicking on naming here 🤓 

I think having a `bodyTemperature` on the `BodyTemperatureComponent` component is kind of _done twice_. I usually name variable corresponding to components without the `Component` prefix, so this could end up in code like this:

```java
BodyTemperatureComponent bodyTemperature = entity.getComponent(BodyTemperatureComponent.class);
if (bodyTemperature.bodyTemperature > 0.5) { ... }
```

If this reflects the state, I'd rather name it something like `current` so that the access reads more like this:

```java
bodyTemperature.current
```